### PR TITLE
Update Animekimi presence

### PR DESCRIPTION
**Pull request by Faelayis**
- Added setting Privacy Mod
- Added setting Show timestamps
- Fixed wrong github source page not found

<details>
<summary>
Preview
</summary>
<br>

Animation
</details>

### CODE REVIEW

File: websites/A/Animekimi/presence.ts [hunk 1]
```diff
@@ -38,12 +38,15 @@ presence.on(
...
-  };
+  const time = await presence.getSetting("timestamps"),
+    privacy = await presence.getSetting("privacy"),
+    presenceData: PresenceData = {
+      largeImageKey: "site",
+      startTimestamp: browsingStamp
+    };
 
   // Presence
+  if (time) presenceData.startTimestamp = browsingStamp;
```
Comment (Bas950) [inline websites/A/Animekimi/presence.ts:49]:
```suggestion
```
unneeded, set at line 45 and removed if !time at line 144

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/A/Animekimi/presence.ts [hunk 2]
```diff
@@ -138,6 +141,22 @@ presence.on("UpdateData", async () => {
...
+    delete presenceData.state;
+  }
+  if (buttons) {
+    presenceData.buttons = [
+      {
+        label: "ดูเว็บไซต์",
+        url: document.location.href.replace(/#\d+/, "")
+      }
+    ];
+  }
```
Comment (SlowLife1661) [inline websites/A/Animekimi/presence.ts:159]: Buttons are not allowed in home page

File: websites/A/Animekimi/presence.ts [hunk 3]
```diff
@@ -138,6 +141,22 @@ presence.on("UpdateData", async () => {
...
     delete presenceData.endTimestamp;
   }
+  if (!time) {
+    delete presenceData.startTimestamp;
+    delete presenceData.endTimestamp;
+  }
+  if (privacy) {
+    delete presenceData.details;
+    delete presenceData.state;
+  }
```
Comment (SlowLife1661) [inline websites/A/Animekimi/presence.ts:151]: Basically show nothing? I wouldn't want it to show nothing even If I want privacy

**Review by Slowlife01 — APPROVED** [review]

File: websites/A/Animekimi/presence.ts [hunk 4]
```diff
@@ -138,6 +156,14 @@ presence.on("UpdateData", async () => {
...
     delete presenceData.endTimestamp;
   }
+  if (!time) {
+    delete presenceData.startTimestamp;
+    delete presenceData.endTimestamp;
+  }
+  if (privacy) {
+    delete presenceData.state;
+    delete presenceData.buttons;
+  }
```
Comment (Bas950) [inline websites/A/Animekimi/presence.ts:166]: ???, privacy mode = disable presence?

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/A/Animekimi/presence.ts (hunk 4 above)
Comment (Bas950) [reply websites/A/Animekimi/presence.ts:166]: I am dome I read it as details, ignore me

**Review by Bas950 — APPROVED** [review]

**Pull request merged**